src/kOS/UserIO/TelnetSingletonServer.cs

```diff
@@ -604,7 +620,7 @@ private void SpawnWelcomeMenu()
             var gObj = new GameObject( "TelnetWelcomeMenu_" + MySpawnOrder, typeof(TelnetWelcomeMenu) );
             Object.DontDestroyOnLoad(gObj);
             welcomeMenu = (TelnetWelcomeMenu)gObj.GetComponent(typeof(TelnetWelcomeMenu));
-            welcomeMenu.Setup(this);
+            welcomeMenu.Attach(this);
```

Comment:
Same method, new name.  Since I now call it more than once in the life of the ``welcomeMenu`` it felt wrong to call it "Setup".